ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -573,9 +573,13 @@ private void loadTaskRuntimes() {
           OptimizingMapper.class,
           mapper -> mapper.selectTaskRuntimes(tableRuntime.getTableIdentifier().getId(), processId));
       Map<Integer, RewriteFilesInput> inputs = TaskFilesPersistence.loadTaskInputs(processId);
+      ArcticTable table = tableManager.loadTable(tableRuntime.getTableIdentifier());
```

Comment:
If `loadTable()` fails, it will cause AMS to fail to start, which is unacceptable.

Reply:
Moved to pollTask